test/tbb/test_limiter_node.cpp

```diff
@@ -605,18 +605,18 @@ TEST_CASE( "Deduction guides" ) {
 }
 #endif
 
+struct TestLargeStruct {
+    char bytes[512]{ 0 };
```

Comment:
Will other element be value initialized?

Reply:
Yes, see [Aggregate initialization](https://en.cppreference.com/w/cpp/language/aggregate_initialization)

> If the number of initializer clauses is less than the number of members and basesor initializer list is completely empty, the remaining members and bases are initialized by their default member initializers, if provided in the class definition, and otherwise copy-initialized from empty lists, in accordance with the usual list-initialization rules (which performs ***value-initialization for non-class types*** and non-aggregate classes with default constructors, and aggregate initialization for aggregates). If a member of a reference type is one of these remaining members, the program is ill-formed.